content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md

```diff
@@ -46,6 +48,22 @@ The fields are described as follows:
 
 * `name`: Name of the Eip object.
 
+* `annotations`:
+
+  * `eip.openelb.kubesphere.io/is-default-eip`: Whether the current Eip object is the default Eip object. The value can be `"true"` or `"false"`. For each Kubernetes cluster, you can set only one Eip object as the default Eip object.
+  
+    When creating a Service, generally you need to add the `lb.kubesphere.io/v1alpha1: openelb`, `protocol.openelb.kubesphere.io/v1alpha1: <mode>`, and `eip.openelb.kubesphere.io/v1alpha2: <Eip name>` annotations to the Service to specify that OpenELB is used as the load balancer plugin, either the BGP, Layer 2, or VIP mode is used, and an Eip object is used as the IP address pool. However, if a default Eip object exists, you do not need to add the preceding annotations to the Service and the system automatically assigns an IP address from the default Eip object to the Service. Detailed rules about IP address assignment are as follows:
+
+    |The Service Uses OpenELB|An Eip Object Is Specified|A Default Eip Obejct Exists|A Common Eip Object Exists|IP Address Assigment|
+    |-|-|-|-|-|
+    |No|No|No|Irrelevant|Pending|
+    |No|No|Yes|Irrelevant|An IP address from the default Eip object|
+    |Yes|No|No|No|Pending|
+    |Yes|No|No|Yes|An IP adderss from a common Eip object|
+    |Yes|No|Yes|Irrelevant|An IP address from the default Eip object|
+     |Yes|Yes|Irrelevant|No|Pending|
+    |Yes|Yes|Irrelevant|Yes|An IP address from the specified Eip object|
+
```

Comment:
@Patrick-LuoYu The format seems not correct, you can preview it on this [link](https://website-av0bn79oa-openelb.vercel.app/docs/getting-started/configuration/configure-ip-address-pools-using-eip/).

<img width="1379" alt="image" src="https://user-images.githubusercontent.com/40452856/164643102-be757d43-ca36-4759-8f73-b2b13bb4571c.png">

Reply:
I previewed the article on my local machine, which showed a correct table.

<img width="1420" alt="image" src="https://user-images.githubusercontent.com/76198553/164960154-ad5aa716-aa30-462d-97cf-c382821525b4.png">
 
Not quite sure if anything of the markdown source code is faulty.

![image](https://user-images.githubusercontent.com/76198553/164960197-70032ddc-23df-40d8-baa6-af49419d82a2.png)